cosmolog/cosmologger.py

```diff
@@ -252,27 +257,18 @@ def _prepare_log_event(self, record):
         })
 
     def format(self, record):
+        super(CosmologgerFormatter, self).format(record)
         e = self._prepare_log_event(record)
         return e.json
 
 
-def _update_config_dict(d, u):
```

Comment:
this is a sad regression :(

Reply:
Not a regression per se, it wasn't actually working correctly, and I've decided it's much nicer to let the advanced user have complete control over how they want to configure python logging, instead of updating it piecewise. Incremental configuration is not really flexible and often confusing.